python/lsst/pex/config/dictField.py

```diff
@@ -259,6 +309,9 @@ def __init__(
         self.dictCheck = dictCheck
         self.itemCheck = itemCheck
 
+    def __get__(self, instance, owner=None, at=None, label="default") -> DictClass[KeyTypeVar, ItemTypeVar]:
+        return super().__get__(instance, owner, at, label)
```

Comment:
Needs overloads.

Reply:
I actually eliminated this method, as typing the `Field` as you indicated above in the base classes linked the proper `TypeVar` to `Dict`

Reply:
Actually I found where this created type problems elsewhere being an overload function with the parent being an overloaded function, things are odd